src/main/java/com/wavefront/AbstractTraceLoader.java

```diff
@@ -68,7 +68,7 @@ private void loadGeneratorConfigurationFile() throws IOException {
     try {
       if (generatorConfig.getGeneratorConfigFile() != null) {
         generatorConfig.initPropertiesFromFile();
-
+// TODO fixme
 //        ObjectMapper mapper = new ObjectMapper()
 //                .registerModule(new ParameterNamesModule())
 //                .registerModule(new Jdk8Module())
```

Comment:
I guess this part could be completely deleted.

Reply:
ok, will do.